lib/matplotlib/colors.py

```diff
@@ -31,6 +31,13 @@
     - k: black
     - w: white
 
+To use the colors that are part of the active color cycle in the
+current style, use a string with the color number in square brackets.
+For example:
+
+    - `[0]`: The first color in the cycle
+    - `[1]`: The second color in the cycle
```

Comment:
Should probably note that negative indexes are not allowed (as evidenced by the regex used).